### PR TITLE
Add Discard Model checks on Invoices table

### PR DESCRIPTION
**Pull request by vipulnsward**
What:
- Add Discard Model checks on Invoices table

Why:
- Invoices data should never be deleted

### CODE REVIEW

**Comment by prasanthchaduvula** [comment]
@vipulnsward Please review it

**Review by vipulnsward — COMMENTED** [review]
Are specs added to all places where kept is used?

**Comment by prasanthchaduvula** [comment]
@vipulnsward specs added to all places where kept is used

File: config/database.yml [hunk 1]
```diff
@@ -3,15 +3,16 @@ default: &default
   encoding: unicode
   host: localhost
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  password:
+  user: <%= ENV.fetch("DB_USER") { "" } %>
```
Comment (vipulnsward) [inline config/database.yml:6]: Need to add `DB_USER`, `DB_PASS` to be added to sample env

Comment (prasanthchaduvula) [reply config/database.yml:6]: @vipulnsward Done

File: .env.example [hunk 2]
```diff
@@ -49,3 +49,7 @@ ENABLE_WEEKLY_REMINDER=
 
 # Sentry
 SENTRY_DSN=
+
+#Database
+DB_USER=
+DB_PASS=
```
Comment (vipulnsward) [inline .env.example:55]: @prasanthchaduvula please fix your editor to avoid these EOF issues

**Pull request merged**